packages/ui/bin/buildStyleVars/transformers.js

```diff
@@ -161,13 +166,32 @@ const lowerHexTransformer = {
     },
 }
 
+
+/**
+ * Transform relative values in percents to em
+ */
+const percentToEmTransformer = {
+    name: 'transformer/percent-to-em',
+    type: 'value',
+    matcher: (token) => {
+        const isTypeMatch = isMatchingType(token, RELATIVE_EM_TYPES)
+        const isPercent = Boolean(typeof token.value === 'string' && token.value.endsWith('%'))
```

Comment:
Why wrap in Boolean()?

Reply:
No needs here, but personal preference. I'll guess I've pasted it from another place and will remove it soon :)

Reply:
Left wrapper for better readability